api/test/unit/controllers/test_search_controller.py

```diff
@@ -15,6 +15,7 @@
 from django_redis import get_redis_connection
 from elasticsearch_dsl import Search
 
+import api.controllers.elasticsearch.helpers
```

Comment:
Nit: using an alias for this would help simplify the code in the tests a bit:

```suggestion
import api.controllers.elasticsearch.helpers as es_helpers
```

Though the alternate form reads more "Python-y" (familiar?) to me:

```suggestion
from api.controllers.elasticsearch import helpers as es_helpers
```


```